playgrounds/react-vite/src/tabs/Collectable.tsx

```diff
@@ -0,0 +1,215 @@
+import { Box, Card, Button, Text } from "@0xsequence/design-system";
+import {
+  useMakeOfferModal,
+  useCreateListingModal,
+  useTransferModal,
+  useSellModal,
+  CollectibleCard,
+  useListListingsForCollectible,
+  useListOffersForCollectible,
+  useCurrencies,
+  useBuyModal,
+} from "@0xsequence/marketplace-sdk/react";
+import { useMarketplace } from "../lib/MarketplaceContext";
+import { useAccount } from "wagmi";
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableHead,
+  TableRow,
+  TableCell,
+} from "./../lib/Table/Table";
+import type { Order } from "@0xsequence/marketplace-sdk";
+
+export function Collectible() {
+  const context = useMarketplace();
+
+  return (
+    <Box paddingTop="3" gap="3" flexDirection="column">
+      <Box gap="3">
+        <Box>
+          <CollectibleCard
+            chainId={Number(context.chainId)}
+            collectionAddress={context.collectionAddress}
+            tokenId={context.collectibleId}
+            onCollectibleClick={() => console.log("Collectible clicked")}
+            onOfferClick={() => console.log("Offer clicked")}
+          />
+        </Box>
+        <Card gap="3"></Card>
+      </Box>
+      <Actions />
+      <ListingsTable />
+      <OffersTable />
+    </Box>
+  );
+}
+
+function Actions() {
+  const context = useMarketplace();
+  const { show: openMakeOfferModal } = useMakeOfferModal();
+  const { show: openCreateListingModal } = useCreateListingModal();
+  const { show: openTransferModal } = useTransferModal();
+  const { address } = useAccount();
+
+  if (!address) {
+    return (
+      <Card gap="3">
```

Comment:
How about let them know what actions they can take? We can keep the message and disable the actions?